index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+</head>
+
+<body>
+
+    <section class="book-programs">
+         <h1>Awesome books</h1>
+        <!--<div class="book">
+            <h4>Lorem ipsuki</h4>
+            <h5>Lorem ipsukm</h5>
+            <button>Remove</button>
+            <hr>
+        </div>
+        <div>
+            <h4>Lorem ipsuki</h4>
+            <h5>Lorem ipsukm</h5>
+            <button>Remove</button>
+            <hr>
+        </div> -->
```

Comment:
- [ ] Please, kindly delete this piece of commented code so the repo becomes cleaner 💪

Reply:
😂 I forgot that one. Thanks.

Reply:
Please! Don't forget to customize your readme @kosher9.  🙏